backend/src/main/java/saci/domain/service/validators/LevelValidator.java

```diff
@@ -0,0 +1,25 @@
+package saci.domain.service.validators;
+
+import lombok.NoArgsConstructor;
+import saci.domain.model.Level;
+
+@NoArgsConstructor
+public final class LevelValidator {
+    public static boolean MaxCoefficientIsValid(Level newLevel) {
+        return newLevel.getMaxCoefficient() > 0 && newLevel.getMaxCoefficient() < 101;
+    }
+
+    public static boolean MinCoefficientIsValid(Level newLevel) {
+        return newLevel.getMinCoefficient() >= 0 && newLevel.getMinCoefficient() < 101;
+    }
+
+    public static boolean OverlappingLevelsValidation(int overlappingLevelsCounter) {
```

Comment:
Same here... this method name is not so clear. 
```suggestion
    public static boolean LevelsAreOverlapping(int overlappingLevelsCounter) {
```